docker/docker-compose.ci.yml

```diff
@@ -48,11 +47,20 @@ services:
     image: unlock
     user: node
     working_dir: /home/unlock/paywall
+    expose:
+      - 3001
+    environment:
+      CI: "${CI}"
+      HTTP_PROVIDER: "${HTTP_PROVIDER}"
+    command: ["npm", "run", "start"]
```

Comment:
Ha! I knew adding a port 3001 to the `unlock` image was probably not solving the problem you were looking to solve ;)

What I am thinking now is that we do not need both `unlock` and `unlock-app`. Maybe worth considering for another PR.